scripts/bsd/openbsd-prep-postgres.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+pkg_add -uvI && \
+pkg_add -I \
+vim--no_x11 git \
+bash \
+git \
+gmake \
+meson \
+pkgconf \
+\
+bison \
+ccache \
+gettext-tools \
+\
+p5-IPC-Run \
+\
+gssdp \
+icu4c \
+libxml \
+libxslt \
+lz4 \
+openpam \
+python%3.8 \
+readline \
+tcl%8.6 \
+\
+login_krb5 \
+openldap-client--gssapi \
+openldap-server--gssapi || true
```

Comment:
What's the deal with the `|| true` here? Won't that end up with the image build succeeding, even though there were package installation failures?